tests/ops/fixtures/saas/square_fixtures.py

```diff
@@ -149,7 +157,6 @@ def create_customer(self, email_address: str, idempotency_key: str) -> Response:
             "birthday": str(faker.date_of_birth()),
             "family_name": faker.first_name(),
             "given_name": faker.last_name(),
-            "phone_number": faker.phone_number(),
```

Comment:
This caused intermittent issues because the generated phone number was not always valid